prototype/views.py

```diff
@@ -53,18 +65,22 @@ def get_context_data(self, *args, **kwargs):
         self.page['has_related_card_item_two'] = has_related_card_item_two
         self.page['has_related_card_item_three'] = has_related_card_item_three
 
-        social_links_builder = SocialLinkBuilder(
-            self.request.build_absolute_uri(),
-            self.page['title'],
-            'Great.gov.uk')
-
         return super().get_context_data(
-            social_links=social_links_builder.links,
+            social_links=self.social_links,
             *args,
-            **kwargs
-        )
+            **kwargs)
 
 
-class NewsArticleDetailView(NewsSectionFeatureFlagMixin, ArticleDetailView):
-    template_name = 'prototype/news_detail.html'
+class NewsArticleDetailView(
+    NewsSectionFeatureFlagMixin,
+    ArticleDetailView,
+):
+    template_name = 'prototype/domestic_news_detail.html'
     url_name = 'news-article-detail'
+
+
+class InternationalNewsArticleDetailView(
+    NewsArticleDetailView, InternationalNewsCMSPageMixin,
+):
+    template_name = 'prototype/international_news_detail.html'
+    url_name = 'international-news-article-detail'
```

Comment:
how is url_name used?

Reply:
i think it was used in `social_links_builder` but isn't anymore. i can probably remove it